pkgs/games/fteqw/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, stdenv
+, pkgs
+, fetchsvn
+, runCommand
+, pkgName ? "fteqw"
```

Comment:
Instead of using this untyped input I would rather move the expression into a common function or file and make the overrides more generic so that the buildFlags are directly supplied for package XY instead of relying on string matches.

Reply:
@SuperSandro2000 I feel like that would make the package a lot less readable, but if you feel it's worth it to be able to define buildFlags, buildInputs, nativeBuildInputs, postFixup, and meta.description elsewhere and have a more common shared derivation, then I will certainly try!